<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family: century gothic;
	font-size: 22px;
}
QFrame
{
	background:#333;
	border-radius:15px;
}
QPushButton
{
	background:red;
	border-radius:60px
}
QToolButton
{
	background:red;
	border-radius:60px
}
QLabel
{
	color:white;
}

QLineEdit
{
background:transparent;
border:none;
color:white;
border-bottom: 1px solid white;
}

QComboBox
{
background:transparent;
border:none;
color:white;
border-bottom: 1px solid white;
}


QPushButton
{
	color:white;
	border-radius:15px;
}
QPushButton:hover
{
	color:red;
	border-radius:15px;
	background:#49ebff
}


</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>451</width>
      <height>461</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="searchSummonerButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>400</y>
       <width>411</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Search Summoner</string>
     </property>
    </widget>
    <widget class="QComboBox" name="regionList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>LOGIN HERE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="summonerEntry">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>401</width>
       <height>51</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true">Username</string>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Downloads/pngegg.png</normaloff>../../Downloads/pngegg.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../Downloads/asd.qrc"/>
 </resources>
 <connections/>
</ui>
